.gitlab/dev_container_deploy/docker_linux.yml

```diff
@@ -178,6 +178,19 @@ dca_dev_master:
     IMG_SOURCES: ${SRC_DCA}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-amd64
     IMG_DESTINATIONS: cluster-agent-dev:master
 
+cws_injector_dev_branch_multiarch:
+  extends: .docker_publish_job_definition
+  stage: dev_container_deploy
+  rules:
+    !reference [.on_all_builds_a7_manual]
```

Comment:
This doesn't seem to account for tags while the 2 jobs are. It shouldn't block the pipelines, but it feels unexpected to build the container and not push it.
Could you clarify the intended configuration?

Reply:
Aren't tags included by the `all builds` part ?

Reply:
Not as such directly (as far as I can see at least)
My understanding is that if the job runs on the `main` branch, we'll start all jobs/builds, which usually also is the case when tagging

Reply:
Discussed offline, since the job is manual no big deal !